src/makeCalculator.js

```diff
@@ -1,10 +1,48 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+
+    multiply(num) {
+      this.result *= num;
+
+      return this;
+    },
+
+    divide(num) {
+      if (num === 0) {
```

Comment:
[CODE KNOWLEDGE] - While it is good to check for division by zero, throwing an error with a generic message such as 'Error: Division by zero' is not very informative. It's better to include more context in the error message, such as the operation that was being attempted and the values involved. For example: 'Cannot divide ' + this.result + ' by zero.'